tools/ci_build/github/azure-pipelines/linux-tvm-ci-pipeline.yml

```diff
@@ -0,0 +1,62 @@
+jobs:
+  - job: Linux_py_TVM_Wheels
+    timeoutInMinutes: 180
+    workspace:
+      clean: all
+    pool: Linux-TVM-CPU
+    steps:
+      - checkout: self
+        clean: true
+        submodules: recursive
+
+      - template: templates/get-docker-image-steps.yml
+        parameters:
+          Dockerfile: tools/ci_build/github/linux/docker/Dockerfile.ubuntu
+          Context: tools/ci_build/github/linux/docker
+          DockerBuildArgs: "--build-arg BUILD_UID=$( id -u )"
+          Repository: onnxruntimecpubuild
+
+      - task: CmdLine@2
+        displayName: 'Install python deps'
+        inputs:
+          script: |
+            python3 -m pip install decorator scipy
```

Comment:
what are these used by? I only see a `docker run` later that does anything substantial. these dependencies won't be in the actual docker image right?

Reply:
see comment below